src/main/java/mate/academy/validator/FieldMatchValidator.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.validator;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+
+    private String firstFieldName;
+    private String secondFieldName;
+    private String message;
+
+    @Override
+    public void initialize(final FieldMatch constraintAnnotation) {
+        firstFieldName = constraintAnnotation.first();
+        secondFieldName = constraintAnnotation.second();
+        message = constraintAnnotation.message();
+    }
+
+    @Override
+    public boolean isValid(final Object value, final ConstraintValidatorContext context) {
+        boolean valid = true;
```

Comment:
It could be simpler 
```suggestion
        Object field = new BeanWrapperImpl(value).getPropertyValue(this. firstFieldName);
        Object fieldMatch = new BeanWrapperImpl(value).getPropertyValue(this. secondFieldName);
        return Objects.equals(field, fieldMatch);
```